### PR TITLE
Add SWITCH edu-ID

### PR DESCRIPTION
**Pull request by tsager**
eduid.ch is used for 2FA for universities in Switzerland

### CODE REVIEW

File: img/e/eduid.ch.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... d="M176,124.5h21.91c12.67,0,18.06,4.59,18.06,17.08v10.83c0,9.55-2.69,18-16.59,18H176Zm23.38,38.38c5.94,0,7.78-5.33,7.78-10.47V141.58c0-7.65-2.51-9.55-9.25-9.55H184.78v30.85Z"/><path class="cls-2" d="M34.24,89.28c0-9.22,4.24-10,12.77-10,7.46,0,12.86.32,12.44,9.72H55.53c-.18-6-1.33-6.27-8.66-6.27-7.65,0-8.71,1-8.71,5.76,0,6.6,3.13,6,9.58,6.23,8.21.46,12.36.23,12.36,9.58,0,9.77-4.61,10.1-13.32,10.1s-13.37-1.06-12.63-11.06h3.92c-.1,7.6,1.38,7.6,8.71,7.6s9.4,0,9.4-6.78c0-6.59-3-5.48-9.27-6C40.09,97.57,34.24,98.4,34.24,89.28Z"/><path class="cls-2" d="M103,114.07H97.1L88.06,83H88L78.8,114.07H73L62.35,79.5h4l9.4,31.57h.09L85.16,79.5h5.76l9.22,31.57h.09l9.63-31.57h4.06Z"/><path class="cls-2" d="M121.93,114.07H118V79.5h3.92Z"/><path class="cls-2" d="M141.19,114.07h-3.92V83h-11.2V79.5h26.32V83h-11.2Z"/><path class="cls-2" d="M165.38,114.35c-6.31,0-10-4-10-10.88V90.1c0-9.77,6-10.87,14.43-10.87H173c7.24,0,9.45,3.13,9.45,10v1.11h-3.92v-1c0-4.84-1-6.64-6.08-6.64h-2.21c-8.11,0-11,1-11,7.29v9.49c0,7.88.32,11.43,6.87,11.43h4.65c6.27,0,8.2-.88,8.2-5.81v-2.76h3.92v3.22c0,7.93-5.07,8.81-12,8.81Z"/><path class="cls-2" d="M215.93,114.07H212V98.17H191.78v15.9h-3.92V79.5h3.92V94.71H212V79.5h3.92Z"/></svg>
```
Comment (Carlgo11) [inline img/e/eduid.ch.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 208.8 169.4"><path fill="#273375" d="M61.4 0h80.8c46.8 0 66.6 17 66.6 63v40c0 35.2-10 66.4-61.2 66.4H61.4Zm86.2 141.5c22 0 28.7-19.6 28.7-38.6V63c0-28.2-9.2-35.2-34-35.2H93.8v113.8Zm-115 27.8H0V0h32.5z"/></svg>
```

File: entries/e/eduid.ch.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "domain": "eduid.ch",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://help.switch.ch/eduid/faqs/?lang=en#mfa",
+    "recovery": "https://help.switch.ch/eduid/faqs/?lang=en#mfa-lost-mobile",
+    "categories": [
+      "universities"
+    ]
```
Comment (Carlgo11) [inline entries/e/eduid.ch.json:15]:
```suggestion
    ],
    "regions": [
      "ch"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @up_the_irons,
I've attached some code suggestions I'd like you to merge.

**Review by tsager — COMMENTED** [review]
Suggested changes look good.

File: img/e/eduid.ch.svg (hunk 1 above)
Comment (Carlgo11) [reply img/e/eduid.ch.svg]: I think you missed this one 😉

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes, it takes longer to show up due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**